src/screens/jobs/new/index.jsx

```diff
@@ -60,6 +63,21 @@ export const NewJobScreen = ({ currentAccount, accountBalance }) => {
     setJobsCategories(orderedCategories)
   }, [response])
 
+  function fetchBnbPrice() {
+    fetch(AVG_PRICE_API_URL)
+      .then((res) => res.json())
+      .then((data) => {
+        setConversionRate(data.price)
+      })
+      .catch(() => {
+        setConversionRate(0)
+      })
+  }
+
+  useEffect(() => {
+    fetchBnbPrice()
+  }, [])
```

Comment:
I think we should add another state variable: `const [isPriceFetched, setIsPriceFetched] = useState(false)`.

1. Add `isPriceFetched` to `[]` in 79 line
2. Add `if (isPriceFetched) return` to 77 line
3. Add `setIsPriceFetched(true)` after `fetchBnbPrice()` call

Why do we need this? To prevent fetching on every state change.